runc.go

```diff
@@ -327,6 +327,7 @@ func (r *Runc) Delete(context context.Context, id string, opts *DeleteOpts) erro
 type KillOpts struct {
 	All       bool
 	ExtraArgs []string
+	RawSignal string
```

Comment:
Not really sure `Raw` is the right prefix here. Signals in Unix are integers and this parameter can take both integers and strings.

Signal or SignalName?

Reply:
Yeah, the naming for this is a little difficult 😅 With `Raw` I was hoping to convey that the signal has potentially not been parsed yet.